xarray/core/missing.py

```diff
@@ -330,19 +332,22 @@ def interp_na(
     interp_class, kwargs = _get_interpolator(method, **kwargs)
     interpolator = partial(func_interpolate_na, interp_class, **kwargs)
 
+    if keep_attrs is None:
+        keep_attrs = _get_keep_attrs(default=False)
```

Comment:
I think the default should be True here? We are just filling in values so it's likely that all attributes are still valid.

Reply:
I guess so, the original option was clearly intended to keep attrs (even though it didn't work).